modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -439,4 +455,82 @@ If no password can be identified, you must reset the admin password via the comm
 
 To change the admin password from the command line, which you can do at any time, follow the guide described in xref:deployment/general/general-info.adoc#password-reset-for-the-admin-user[Password Reset for the Admin User].
 
+== Volume Migration
+
+This section gives some guidance if you want to migrate the Infinite Scale docker internal volumes to docker volumes using a local path. This can be required like to separate the container from its data or if a high data volume is expected. See additional documentation in the xref:deployment/tips/useful_mount_tip.adoc[Start a Service After a Resource is Mounted] if you want to use network mounts like NFS or iSCSI for the data directory.
+
+* Prepare two directories which will provide the mount point for Infinite Scale `data` and `config`. +
+The example will use the local path `/mnt/data` and `mnt/config`, adapt according your environment.
+
+* For the following steps, the compose setup _must be_ in `up` state, the containers must provide a container ID for copying.
+
+** Stop the running instance. By doing so, the instance gets stopped but containers are not removed compared when downing it:
```

Comment:
```suggestion
** Stop the running instance. By doing so, the instance gets stopped but containers are not removed compared to when downing it:
```

The "up" and "down" commands are causing us to manufacture words like "downing". But I suppose that is OK for a technical document. Otherwise, we have to write many more words each time. So I think we keep using "downing" (and I suppose "upping the instance" will be written somewhere also).

Reply:
As long we don't _drown_ the container... 😅